coordinator-service/src/app.ts

```diff
@@ -89,6 +127,36 @@ export function initExpress({
         },
     )
 
+    app.get('/chunks/:id', (req, res) => {
```

Comment:
Just to clarify - is anyone using this method anymore?

Reply:
No, this can be removed now.